task_6/map-practice.js

```diff
@@ -0,0 +1,9 @@
+var mapPractice=function(niz){
+    
+    for (let i = 0; i < niz.length; i++) {
```

Comment:
Poenta ovog zadatka je bila da koristis ugradjenu map metodu JavaScript-a
Sa map to bi moglo recimo ovako da se uradi
```suggestion
   return niz.map((num, i) => num - i);
```